moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp

```diff
@@ -37,7 +37,7 @@
 #include <cmath>
 #include <moveit/pointcloud_octomap_updater/pointcloud_octomap_updater.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_monitor.h>
-#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_eigen/tf2_eigen.h>
```

Comment:
I think this change makes the dependency on `tf2_geometry_msgs` obsolete. Please remove in package.xml.

Reply:
It seems there are still dependencies in other subfolders of the package, such as the depthmap updater.